aixtend/modules/model.py

```diff
@@ -48,33 +47,22 @@ def __init__(
             id (str): ID of the Model
             name (str): Name of the Model
             supplier (str): supplier of the Model
-            api_key (str, optional): API key of the Model. Defaults to None.
+            api_key (str): Team API Key to run the model.
```

Comment:
What is the behaviour when?
1.  Team key is not present
2.  Team key is present and wrong
We just need to test these out.

Reply:
It means that if its not supplied as a parameter.

Reply:
If Team Key is incorrect, we get an error from BE and raise an exception here : https://github.com/aixplain/aiXtend/blob/v2_init_release_changes/aixtend/factories/model_factory.py#L73

Reply:
`Status 401: Model Creation: Status 401 - Unauthorized`
We get this if the API Key is not set. I can add a separate warning for this to make it more explicit